### ISSUE TITLE
[SAP][IllustratedMessage]: `aria-label` set to `undefined` for `svg` and being read out as `undefined graphic` by JAWS

### ISSUE THREAD

**Issue by rama1990br** [SAP/ui5-webcomponents#6839]
### Describe the bug

Instance:
REMOVED - see internal slack channel

**Steps:**
Login to instance.
Directly go to the link given
Script-Based Integration Settings page is opened.
Navigate on page in Virtual PC mode/Browse mode.

Actual result:
The  graphic is being read out as 'undefined' and is not hidden from screen readers.

Since this image serves no additional meaningful purpose this graphic should be considered as decorative and hidden from screen readers. However, using `aria-hidden` does not work in this case since the title and subtitle associated with the illustrated message won't be read out.

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Decorative images do not present important content and therefore decorative images should have empty `aria-label` or `role="presentation"`. Decorative images must be hidden from screen readers.

### Screenshots or Videos

![JAWS history](https://user-images.githubusercontent.com/32254036/229023850-6dba047d-6bae-4bb3-a46d-b459bb8594d0.png)
![decorative image with aria-label](https://user-images.githubusercontent.com/32254036/229023852-55c92ffd-e011-411b-a9c3-44d199a87ecb.png)

### UI5 Web Components for React Version

1.5.1

### UI5 Web Components Version

1.9.1

### Browser

Chrome, Firefox

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @rama1990br,

Is the issue reproducible in the playground samples:
https://sap.github.io/ui5-webcomponents/playground/components/IllustratedMessage/

I wasn't able to reproduce it.

In case its not reproducible in the playground could you please develop a simple snippet like the following:
https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html

Best regards,
Boyan
